packages/@adobe/spectrum-css-temp/components/tabs/skin.css

```diff
@@ -82,6 +114,10 @@ governing permissions and limitations under the License.
 
   .spectrum-Tabs-selectionIndicator {
     background-color: var(--spectrum-tabs-quiet-selection-indicator-color);
+
+    &.spectrum-Tabs--emphasized {
```

Comment:
i don't think this is the selector you want, it'll generate something like
`.spectrum-Tabs .spectrum-Tabs-selectionIndicator.spectrum-Tabs--emphasized`
double hyphen means state, single means hierarchy/namespace
I'd expect this to be
`.spectrum-Tabs.spectrum-Tabs--emphasized .spectrum-Tabs-selectionIndicator`
you may need to re-arrange some of this to accomplish that, ampersand just means, take the selector (computed) above and insert it in place of the ampersand

Reply:
thanks! refactored